### ISSUE TITLE
tkinter gui develop

### ISSUE THREAD

**Issue by AustrianOakvn** [AustrianOakvn/GIS81507#9]

**Comment by damtien444** [comment]
I think we should merge the Twitch dev and Game dev branches to do this functionality. Because some features of the GUI is from both team.

**Comment by AustrianOakvn** [comment]
yeah sure, you can choose to merge into the branch that we are gonna work on whether it is game dev branch or twich dev branch all looks good to me.

**Comment by damtien444** [comment]
okay, I created the [gui](https://github.com/AustrianOakvn/GIS81507/tree/gui) branch containing code from both.
